release/src/module_dependencies/main.py

```diff
@@ -0,0 +1,172 @@
+import urllib.request
+import json
+import base64
+import networkx as nx
+
+# Sorts the ballerina standard library module list in ascending order
+def sortModuleNameList():
+    with open('./release/resources/module_list.json') as f:
+          nameList = json.load(f)
+
+    nameList['modules'].sort()
+    
+    with open('./release/resources/module_list.json', 'w') as jsonFile:
+        jsonFile.seek(0) 
+        json.dump(nameList, jsonFile, indent=4)
+        jsonFile.truncate()
+        
+    return nameList['modules'] 
+
+# Gets dependencies of ballerina standard library module from build.gradle file in module repository
+# returns: list of dependencies
+def getDependencies( balModule ):
+
+    data = urllib.request.urlopen("https://raw.githubusercontent.com/ballerina-platform/" + balModule + "/master/build.gradle")
+
+    dependencies = []
+
+    for line in data:
+
```

Comment:
Extra empty line? Check for other similar cases as well.

Reply:
Fixed